app/assets/javascripts/application.js

```diff
@@ -58,3 +58,4 @@
 //= require noty_notification.js
 //= require jquery-confirm/js/jquery-confirm.js
 //= require jquery.atwho
+//= require emoji.js
```

Comment:
I think I'd prefer not to include the entire emoji.js file here, and to leave it in the `/lib/` folder. These files are compiled together, and I think we can treat the emoji file as a static asset, so the system doesn't have to process it (since it's so big!). But we should include `comment-autocomplete.js` here -- i actually don't see that one in the PR, did you forget to add it?

Reply:
Hi @jywarren I think that I did a poor job naming the emoji script, in that it is named the same as the file that held the emoji json. In this version of the code, emoji.js is the script living in app/assets/javascripts/emoji.js that includes within it the defined emojis (not being read from public, just defined in a var), and deleted the file from public/lib/emoji.js since it would not be needed. I can remove the require statement from application.js.

 I'm not sure about the comment-autocomplete.js? I had not done anything with that, when I  add it in to application.js I get the error ```couldn't find file 'comment-autocomplete.js' with type 'application/javascript'``` so perhaps I need to fetch more recent changes and do a rebase? I believe I have the config/database.yml.sqlite removed properly now so once I clear up the details above, I'll resubmit.

Thanks again for all of your help! And the callout in the community checkin, it is really wonderful to have the opportunity to work with you on this :)